components/tidb_query_vec_expr/src/types/expr.rs

```diff
@@ -25,24 +53,29 @@ pub enum RpnExpressionNode {
     },
 
     /// Represents a reference to a column in the columns specified in evaluation.
-    ColumnRef { offset: usize },
+    ColumnRef {
+        offset: usize,
+    },
+    // Represents a operator call
+    Operator {
```

Comment:
Aggregations cannot be nested and must be the most outside operator, so there is no need to introduce a new variant for the Rpn node

Reply:
ditto